components/raftstore/src/coprocessor/mod.rs

```diff
@@ -222,6 +222,16 @@ pub trait ApplySnapshotObserver: Coprocessor {
     fn should_pre_apply_snapshot(&self) -> bool {
         false
     }
+
+    // Hook when apply snapshot is committed on disk.
```

Comment:
Please elaborate the meaning of "committed", it may cause confusion because Raft also use term "commit".

Reply:
Changed to
```
    // Hook when apply snapshot is ingested, and the state has been changed to Normal and persisted.
    // The snapshot will not be re-iningested after the restart if this hook is called.
```